versions.tf

```diff
@@ -22,6 +22,11 @@ terraform {
       source  = "hashicorp/google"
       version = ">= 3.53, < 5.0"
     }
+
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = ">= 3.53, < 5.0"
+    }
   }
 
   provider_meta "google" {
```

Comment:
Lets also add a meta block for the beta provider.
```
provider_meta "google-beta" {
    module_name = "blueprints/terraform/terraform-google-lb-internal/v4.5.0"
  }
  ```

Reply:
Added, thanks!